user-guide/modules/ROOT/nav.adoc

```diff
@@ -1,11 +1,19 @@
 * xref:intro.adoc[]
-* xref:getting-started.adoc[]
 * xref:whats-inside.adoc[]
-* xref:reporting-issues.adoc[]
-* xref:advanced-builds.adoc[]
-* xref:faq.adoc[]
-* xref:tutorials.adoc[]
-* xref:testing-debugging.adoc[]
+* xref:getting-started.adoc[]
+
+* Building 
+** xref:installing-select-libraries.adoc[]
+** xref:advanced-builds.adoc[]
+
+* Using
+** xref:reporting-issues.adoc[]
+** xref:faq.adoc[]
+** xref:tutorials.adoc[]
+** xref:testing-debugging.adoc[]
+
+* Tutorials
+** xref:boost-test-tutorial.adoc[]
 
 * Community
 ** xref:bsl.adoc[]
```

Comment:
There's good blog post by pdimov where he explains the motivation behind the boost license. We can use this content here.

This needs more pages or it can just go one level up.

Reply:
@alandefreitas I am anticipating the Tutorials section will grow considerably. Agreed the license stuff can be added.

Reply:
@alandefreitas @anarthal I have added pdimov's blog and the BSL, and addressed all your comments in the "user guide structure feedback" commit.

Reply:
Looks great :+1: